keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -146,6 +153,16 @@ In its first implementation the risks are low as it tries to be a CRI API
 change with minimal changes to the kubelet and it is gated by the feature
 gate `ContainerCheckpoint`.
 
+One possible risk that was identified during Alpha is that the disk of
+the node requesting the checkpoints could fill up if too many checkpoints
+are created. One approach to solve this was some kind of garbage collection
+of checkpoint archives. A pull request to implement garbage collection
+was opened ([#115888](https://github.com/kubernetes/kubernetes/pull/115888))
+but during review it became clear that the kubelet might not be the right
+place to implement checkpoint archive garbage collection and the pull request
+was closed again. Currently the most likely solution seems to be to implement
+the garbage collection in an operator.
+
```

Comment:
outside the diff, but 

>A kubelet API to trigger the checkpointing of a container will be
introduced as described in [Implementation](#implementation).
introduced as described in [Implementation](#implementation).
Also see https://github.com/kubernetes/kubernetes/pull/104907 for details.
Also see https://github.com/kubernetes/kubernetes/pull/104907 for details.

should be expanded in this KEP so that readers can understand the API that exists without reading the kubelet code.

Reply:
It is expanded in the `#Implementation` section of the KEP. Is that enough? Not sure if you saw it, but `Implementation` is a link to a section of the KEP.

Do you think more details are required in the `Design Details` section. I kind of remember that the current layout of `Implementation` and `Design Details` was the result of a previous review. Especially the link from one section to another.

Reply:
I guess I am too deep into the topic. Let me add something concerning the high level flow:

"Triggering the kubelet API endpoint will trigger the newly added `CheckpointContainer` CRI RPC"

Reply:
> I guess I am too deep into the topic. Let me add something concerning the high level flow:
> 
> "Triggering the kubelet API endpoint will trigger the newly added `CheckpointContainer` CRI RPC"

KEPs will usually include
1. the URL being hit
2. the permissions required to be authorized for the URL
3. which class of user (rbac clusterrole) will grant access
4. rough type definition of the new API, with explanations of each field
5. if expected latency isn't simply "time to write to etcd" an explanation of how long it will take

Reply:
Thanks. Added.